scheds/rust/scx_layered/src/main.rs

```diff
@@ -3157,6 +3181,27 @@ fn verify_layer_specs(specs: &[LayerSpec]) -> Result<()> {
                             bail!("Spec {:?} has too long a process name prefix", spec.name);
                         }
                     }
+                    LayerMatch::HintEquals(hint) => {
+                        if *hint > 1024 {
+                            bail!(
```

Comment:
Should this bail or just warn? I guess maybe it's good to fail fast, but ignoring would seem fine as well.

Reply:
The lookup table is sized to 1025 entries, so not warning would just mean we silently ignore invalid settings, and do not pick up the hint at runtime for the layer.

Reply:
Ahh that makes sense, thanks!